container-onnxruntime-bundle/src/main/java/ai/onnxruntime/package-info.java

```diff
@@ -0,0 +1,8 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+/**
+ * @author arnej
+ */
+@ExportPackage
+package ai.onnxruntime;
```

Comment:
Why is this package exported? It does not seem to contain any code.

Reply:
that's just seemingly, it exports the actual onnruntime classes.